eldritch/ancient_ones.py

```diff
@@ -0,0 +1,147 @@
+import abc
+
+from eldritch import events, places
+from eldritch.monsters import Monster
+
+
```

Comment:
AncientOne should also(?) probably be a subclass of mythos.GlobalEffect to get all the default triggers, interrupts, and modifiers.

Reply:
Did you mean to go back and fix this?